source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,89 @@
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
+
+**Participant**
+
+In the context of Playbooks, participants belong to or take part in a run. They can be a member or a Playbook Admin.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility as well as functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook. They do not have access to the System Console and their privileges are managed by the System Admin.
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view Public and Private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs. They also can't create Public or Private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+You can also set permissions for read-only playbooks that do allow members to create new public or private playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Then, check **Create Public Playbook** and **Create Private Playbook**.
+4. Select **Save**.
+
+Manage how playbooks are accessed
+---------------------------------
+
+A Public playbook and its associated runs can be viewed by any member of the team. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
+
+Restrict who can convert playbooks from Public to Private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can control whether Members can convert playbooks from public to private.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from Public to Private:
+
+1. Go to **System Console > User Management > Permissions**.
+2. Scroll down to **Playbook Admin**.
+3. Uncheck **Convert Playbooks**.
```

Comment:
Wouldn't this prevent Playbook Admins from converting playbook, instead of allowing them to do so?

Reply:
Yep you're right. Will rectify.